### PR TITLE
Ignore loading errors during value class preload

### PR DESCRIPTION
**Pull request by theresa-m**
Fixes: https://github.com/eclipse-openj9/openj9/issues/19119

### CODE REVIEW

File: runtime/vm/createramclass.cpp [hunk 1]
```diff
@@ -1954,8 +1954,7 @@ loadFlattenableFieldValueClasses(J9VMThread *currentThread, J9ClassLoader *class
 
 			U_8 *preloadClassName = J9UTF8_DATA(preloadClassNameUtf8);
 			U_16 preloadClassLength = J9UTF8_LENGTH(preloadClassNameUtf8);
-
-			internalFindClassUTF8(currentThread, preloadClassName, preloadClassLength, classLoader, classPreloadFlags);
+			internalFindClassUTF8(currentThread, preloadClassName, preloadClassLength, classLoader, J9_FINDCLASS_FLAG_EXISTING_ONLY);
```
Comment (hangshao0) [inline runtime/vm/createramclass.cpp:1957]: `J9_FINDCLASS_FLAG_EXISTING_ONLY` won't trigger a class to be loaded. It only searches for the class in the existing loaded classes in the class table. The flag should be `classPreloadFlags & ~J9_FINDCLASS_FLAG_THROW_ON_FAIL`.

Comment (hangshao0) [reply runtime/vm/createramclass.cpp:1957]: For null restricted field types (or Q types), any class loading exception should still be thrown. This block of code handling preload attribute here can be moved after the following while loop that exams each field (probably right before `done:`).

Comment (theresa-m) [reply runtime/vm/createramclass.cpp]: Thanks. I have updated my change.

File: runtime/vm/createramclass.cpp [hunk 2]
```diff
@@ -2079,6 +2064,22 @@ loadFlattenableFieldValueClasses(J9VMThread *currentThread, J9ClassLoader *class
 	if (NULL != superClazz) {
 		*valueTypeFlags |= (J9ClassHasReferences & superClazz->classFlags);
 	}
+
```
Comment (hangshao0) [inline runtime/vm/createramclass.cpp:2067]: Could you also update the doxygen comment of this function to mention the new behaviour ?

Comment (theresa-m) [reply runtime/vm/createramclass.cpp:2070]: Updated.

**Comment by hangshao0** [comment]
Jenkins test sanity,extended winval jdknext

**Comment by hangshao0** [comment]
Jenkins test sanity,extended xlinuxval jdknext

**Pull request merged**